app/controllers/Checkout.scala

```diff
@@ -417,7 +417,7 @@ class Checkout(fBackendFactory: TouchpointBackends, commonActions: CommonActions
       // should a client want not to proxy via this server.
       Ok(Json.obj("Addresses" -> result.Addresses))
     } recover {
-      case error if error.getMessage == "Bad Request" =>
+      case error if error.getMessage == "Bad Request" || error.getMessage == "Not Found" =>
```

Comment:
This error message came through in the logs this morning...

Reply:
(It's not what I saw when testing locally, I guess getAddress make a distinction between a garbage string and something which looks like a postcode, but isn't actually a postcode)